samples/boards/stm32/power_mgmt/blinky_standby_shutdown_mode/sample.yaml

```diff
@@ -0,0 +1,15 @@
+sample:
+  name: STM32 GPIO Power Management
+tests:
+  sample.boards.stm32.power_mgmt.blinky_standby_shutdown_mode:
+    platform_allow: nucleo_l476rg
+    tags: LED power
+    harness: console
+    harness_config:
+      type: one_line
+      regex:
+        - "Device ready"
```

Comment:
This will fail in CI. 
First message when running twister is likely `Reset cause: Reset pin`

Reply:
ok